cmd/nerdctl/image_convert.go

```diff
@@ -188,52 +219,52 @@ func imageConvertAction(cmd *cobra.Command, args []string) error {
 				return err
 			}
 			convertType = "zstdchunked"
+		case overlaybd:
+			obdOpts, err := getOBDConvertOpts(cmd)
+			if err != nil {
+				return err
+			}
+			obdOpts = append(obdOpts, overlaybdconvert.WithClient(client))
+			obdOpts = append(obdOpts, overlaybdconvert.WithImageRef(srcRef))
+			convertFunc = overlaybdconvert.IndexConvertFunc(obdOpts...)
+			convertOpts = append(convertOpts, converter.WithIndexConvertFunc(convertFunc))
+			convertType = "overlaybd"
+		case nydus:
+			nydusOpts, err := getNydusConvertOpts(cmd)
+			if err != nil {
+				return err
+			}
+			convertHooks := converter.ConvertHooks{
+				PostConvertHook: nydusconvert.ConvertHookFunc(nydusconvert.MergeOption{
+					WorkDir:          nydusOpts.WorkDir,
+					BuilderPath:      nydusOpts.BuilderPath,
+					FsVersion:        nydusOpts.FsVersion,
+					ChunkDictPath:    nydusOpts.ChunkDictPath,
+					PrefetchPatterns: nydusOpts.PrefetchPatterns,
+				}),
+			}
+			convertOpts = append(convertOpts, converter.WithIndexConvertFunc(
+				converter.IndexConvertFuncWithHook(
+					nydusconvert.LayerConvertFunc(*nydusOpts),
+					true,
+					platMC,
+					convertHooks,
+				)),
+			)
+			convertType = "nydus"
 		}
-		convertOpts = append(convertOpts, converter.WithLayerConvertFunc(convertFunc))
 
+		if convertType != "overlaybd" {
+			convertOpts = append(convertOpts, converter.WithLayerConvertFunc(convertFunc))
+		}
 		if !oci {
-			logrus.Warnf("option --%s should be used in conjunction with --oci", convertType)
+			logrus.Warnf("option --%s should be used in conjunction with --oci, forcibly enabling on oci mediatype for %s conversion", convertType, convertType)
```

Comment:
"forcibly enabling on oci mediatype" is untrue for estargz.
True for Nydus. Not sure for overlaybd.

Reply:
Sorry, it was fixed in [here](https://github.com/containerd/nerdctl/pull/1517/files#diff-54ec048a1d59c912fc54d9a2463beb5f84676ef7a2f93cebcbd9dced0cda9027R261-R265), `mediatype` with/without `--oci` has been confirmed.